tests/visco_plastic_2_phases_compositions_lookups.prm

```diff
@@ -0,0 +1,122 @@
+# Copy of visco_plastic_phases.prm, but include multiple
```

Comment:
During the next round of revisions, can you add a few sentences about the expected output relative to the (to be added) test results?